Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabPublicInterface.h

```diff
@@ -149,14 +149,6 @@ namespace AzToolsFramework
              * @return Returns true if the prefab has unsaved changes, false otherwise. If path is invalid, returns an error message.
              */
             virtual PrefabRequestResult HasUnsavedChanges(AZ::IO::Path prefabFilePath) const = 0;
-
-            /**
-             * [DEPRECATION]--This function is marked for deprecation. Please use DeleteEntitiesAndAllDescendantsInInstance instead.
-             * Deletes all entities from the owning instance. Bails if the entities don't all belong to the same instance.
-             * @param entities The entities to delete.
-             * @return An outcome object; on failure, it comes with an error message detailing the cause of the error.
-             */
-            virtual PrefabOperationResult DeleteEntitiesInInstance(const EntityIdList& entityIds) = 0;
```

Comment:
This shouldn't be clubbed with this PR because deprecation PRs should be easy to find and reversed if needed by users. Please remove these changes from this PR and create a follow up one. Also, this should go to stabilization branch imo